src/main/webapp/js/login.js

```diff
@@ -25,6 +25,75 @@ function login() {
     })
 }
 
+function collectDto(){
+    var usernameTxt = document.getElementById("username-txt");
+    var password1Txt = document.getElementById("password1-txt");
+    var password2Txt = document.getElementById("password2-txt");
+    var pwd1 = password1Txt.value;
+    var pwd2 = password2Txt.value;
+    if (pwd1 !== pwd2) {
+        showError("Passwords doesn't match!")
+        return;
+    }
+    var dto = {
+        "username": usernameTxt.value,
+        "password": pwd1
+    }
+    register(dto)
```

Comment:
Please don't leave out semicolons at the end of statements, otherwise weird things may happen in some cases.